docs/using_plugins.rst

```diff
@@ -195,18 +193,33 @@ If your API uses `method-based dispatching <http://flask.pocoo.org/docs/0.12/vie
 By default, apispec only knows how to set the type of
 built-in marshmallow fields. If you want to generate definitions for
 schemas with custom fields, use the
-`apispec.ext.marshmallow.swagger.map_to_swagger_type` decorator:
+`apispec.ext.marshmallow.openapi.OpenAPIConverter.map_to_swagger_type` decorator
+accessible through the marshmallow plugin instance:
 
 .. code-block:: python
 
-    from apispec.ext.marshmallow import swagger
+    from apispec import APISpec
+    from apispec.ext.marshmallow import MarshmallowPlugin
     from marshmallow.fields import Integer
 
-    @swagger.map_to_swagger_type('string', 'uuid')
+    ma_plugin = MarshmallowPlugin()
+
+    spec = APISpec(
+        title='Gisty',
+        version='1.0.0',
+        openapi_version='2.0',
+        info=dict(
+            description='A minimal gist API'
+        ),
+        plugins=[ma_plugin]
+    )
+
+
+    @ma_plugin.openapi.map_to_swagger_type('string', 'uuid')
```

Comment:
This breaks the [Law of Demeter](https://www.wikiwand.com/en/Law_of_Demeter) by accessing a method on an internal attribute. Also, it's a lot of typing. Perhaps we should expose `map_to_swagger_type` on `MarshmallowPlugin`?

Reply:
Done.